syntax = "proto3";

package helloworld;

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's full name.
message HelloRequest {
  string user_full_name = 1; // More descriptive than "name"
}

// The response message containing the greeting message.
message HelloReply {
  string greeting_text = 1; // More descriptive than "message"
}
